docs/messages.rst

```diff
@@ -84,19 +84,19 @@ Packaging
 
 Finally, you must distribute your schema to clients. It is recommended that you
 maintain your message schema in your application's git repository in a separate
-Python package. The package name should be ``<your-app-name>_schema``.
+Python package. The package name should be ``<your-app-name>_messages``.
```

Comment:
Can we please suggest `-` instead of `_`?  Package names with underscores look ugly :-)

Reply:
FWIW, we used `python3-copr-messaging` name, but we can add the provide (and in future split the package into two parts).

Reply:
I agree it looks ugly, but I think there is (or was?) a bug in pip that made it treat differenly packages with a dash and with an underscore. I'll check that and see if it's still valid.

Don't worry about renaming, it's not a hard convention, just a suggestion for new packages.

Reply:
> I agree it looks ugly, but I think there is (or was?) a bug in pip that made it treat differenly packages with a dash and with an underscore. I'll check that and see if it's still valid.

The problem I'm only aware of is that a python module name can not contain
dashes -- so you can `import some.thing.some_where`.

But the package (project) name and the python module name don't have to be
the same.